host-controller/src/main/java/org/jboss/as/domain/controller/transformers/ManagementTransformers.java

```diff
@@ -58,26 +60,33 @@ static ChainedTransformationDescriptionBuilder buildTransformerChain(ModelVersio
         // Discard the domain level core-service=management resource and its children unless RBAC is enabled
         // Configuring rbac details is OK (i.e. discarable), so long as the provider is not enabled
         ChainedTransformationDescriptionBuilder chainedBuilder = TransformationDescriptionBuilder.Factory.createChainedInstance(CoreManagementResourceDefinition.PATH_ELEMENT, currentVersion);
-        ResourceTransformationDescriptionBuilder builder = chainedBuilder.createBuilder(currentVersion, DomainTransformers.VERSION_1_8);
-        builder.setCustomResourceTransformer(new ResourceTransformer() {
-            @Override
-            public void transformResource(ResourceTransformationContext context, PathAddress address, Resource resource) throws OperationFailedException {
-                Resource accessAuth = resource.getChild(AccessAuthorizationResourceDefinition.PATH_ELEMENT);
-                if (accessAuth != null) {
-                    ModelNode model = accessAuth.getModel();
-                    String providerName = AccessAuthorizationResourceDefinition.PROVIDER.resolveModelAttribute(ExpressionResolver.REJECTING, model).asString();
-                    AccessAuthorizationResourceDefinition.Provider provider = AccessAuthorizationResourceDefinition.Provider.valueOf(providerName.toUpperCase(Locale.ENGLISH));
-                    if (provider != AccessAuthorizationResourceDefinition.Provider.SIMPLE) {
-                        // Continue to lower levels where this will be rejected
-                        final ResourceTransformationContext childContext = context.addTransformedResource(PathAddress.EMPTY_ADDRESS, resource);
-                        childContext.processChildren(resource);
-                    }
-                }
+        ResourceTransformationDescriptionBuilder builderCurrentTo41 = chainedBuilder.createBuilder(currentVersion, DomainTransformers.VERSION_4_1);
+        builderCurrentTo41.addChildResource(AccessAuthorizationResourceDefinition.PATH_ELEMENT)
+                .addChildResource(AccessConstraintResources.SENSITIVITY_PATH_ELEMENT)
+                .addChildResource(SensitivityClassificationTypeResourceDefinition.PATH_ELEMENT)
+                //todo should we really just discard it?
+                .discardChildResource(PathElement.pathElement(SensitivityResourceDefinition.PATH_ELEMENT.getKey(), SensitivityClassification.SERVER_SSL.getName()))
+                .build();
```

Comment:
What led to this? AFAIK there should be no need to transform anything related to RBAC from master to 4.1. These resources have been around since we first did RBAC.